src/components/accounts/popup.js

```diff
@@ -0,0 +1,128 @@
+                <Row>
+                    <Col xs={12}>
+                        <h1>Add Ether</h1>
+                        <FlatButton
+                            label="Cancel"
+                            primary={true}
+                            onTouchTap={handleCloseAccountDialog}
+                            icon={<FontIcon className="fa fa-close" />}
+                          />
+                    </Col>
+                </Row>
+                <Row>
+                    <Col xs={8}>
+                        <DescriptionList>
+                            {account.get('description') && <div>
+                            <DescriptionTitle>{t('show.description.title')}</DescriptionTitle>
+                            <DescriptionData>{account.get('description')}</DescriptionData>
+                            </div>}
+
+                            <DescriptionTitle>{t('show.description.address')}</DescriptionTitle>
+                            <DescriptionData>{account.get('id')}</DescriptionData>
+
+                            <DescriptionTitle>{t('show.description.sentTransactions')}</DescriptionTitle>
+                            <DescriptionData>{account.get('txcount') || '0'}</DescriptionData>
+
+                            <DescriptionTitle>{t('show.description.etherBalance')}</DescriptionTitle>
+                            <DescriptionData>{value}</DescriptionData>
+                            <DescriptionData>{value} {pending}</DescriptionData>
+
+                            <DescriptionTitle>{t('show.description.tokenBalance')}</DescriptionTitle>
+                            <DescriptionData>
+                                {account
+                                    .get('tokens')
+                                    .map((tok) =>
+                                        <TokenRow token={tok} key={tok.get('address')}/>
+                                )}
+                            </DescriptionData>
+
+                            <DescriptionTitle>Equivalent Values</DescriptionTitle>
+                            <DescriptionData>
+                                <div><span>BTC {fiat.btc}</span></div>
+                                <div><span>USD {fiat.usd}</span></div>
+                                <div><span>CNY {fiat.cny}</span></div>
+                            </DescriptionData>
+
+                        </DescriptionList>
+                    </Col>
+                    <Col xs={4} md={2} mdOffset={2}>
+                        <QRCode value={account.get('id')} />
+                    </Col>
+                </Row>
+            </Dialog>
+    );
+});
+
+Render.propTypes = {
+    account: PropTypes.object.isRequired,
+    handleCloseAccountDialog: PropTypes.func.isRequired,
+    open: PropTypes.bool.isRequired,
+};
+
+const AccountPopup = connect(
+    (state, ownProps) => {
+        const account = state.get('accountPopup', Immutable.fromJS());
+        const open = account === {};
```

Comment:
It looks like you are fetching `accountPopup` from the global state, which isn't necessary. Since the `ADD ETC` button is in the account page, you can just pass the `account` in as a prop. eg:
```
<AccountPopup account={ownProps.account} />
``` 
(assuming that is what you are trying to do)